src/components/Home.js

```diff
@@ -1,4 +1,5 @@
 import React from "react";
+// import axios from "axios";
```

Comment:
commented code

Reply:
this was not a file i was working on but i am fixing that